src/lib/firebase.js

```diff
@@ -16,5 +16,6 @@ const config = {
 
 // Initalize Firebase.
 const fb = firebase.initializeApp(config);
+const db = firebase.firestore();
 
-export { fb };
+export { fb, db };
```

Comment:
Do we need to add this? If you need access to `firestore`, you can use `withFirebase` in your component:

```jsx
function SomeComponent(props) {
  const stuff = props.firebase.collection('items').map(renderStuff)

  return <>Hello</>
}

export default withFirebase(SomeComponent)
```

Reply:
So 'withFirebase' is a built in function of React?

Reply:
`withFirebase` is what’s called a “higher order component” (HOC) and is provided by `react-firestore` as a way to directly use the Firestore APIs. I’m on my phone, but hopefully that’s enough info to help you google more info about how to use it!